### ISSUE TITLE
Co-registration between input channels

### ISSUE THREAD

**Issue by cmulshine** [MIC-DKFZ/nnUNet#2271]
In the dataset_format documentation, it is specified that "The different input channels MUST have the same geometry (same shape, spacing (if applicable) etc.) and must be co-registered (if applicable)." 

I'm hoping to use misalignment augmentation to improve the effectiveness of image registration between multiple MRI modalities (meaning the training images are not currently registered between input channels). Is that an issue?

**Comment by cmulshine** [comment]
On a related note, the example used in the 'dataset_format' documentation includes 4 input channels in the 'imagesTr' folder: FLAIR (0000), T1w (0001), T1gd (0002) and T2w (0003); but only one image in the 'labelsTr' folder. Want to confirm that this implies the labelsTr folder should only include the ground truth/mask for the fixed image (and not the masks for the moving images)? Thanks!

**Comment by saikat-roy** [comment]
Hey @cmulshine. Sorry for the late reply. The recommendation is the most common and of course the most ideal situation. The misalingment augmentation of course means that you are taking control of the spacing and alignment, so the results are of course contingent on what you do. The images still need the shape of course for pytorch to stack them channel-wise. The labels in general have the same geometry as the input - in your case, you have to decide what you want to segment. It should probably be the fixed image in your case.
